javalib/src/main/scala/java/util/Formatter.scala

```diff
@@ -631,7 +662,7 @@ final class Formatter(private[this] var dest: Appendable)
   }
 
   private def applyUpperCase(flags: Flags, str: String): String =
-    if (flags.upperCase) str.toUpperCase
+    if (flags.upperCase) str.toUpperCase() // not localized
```

Comment:
I'm not sure I understand this comment: In other places in this file, "not localized" seems to refer to "*should* not be localized". Here, I'm unsure (the JavaDoc seems contradictory to me):

> are the same as those for the corresponding lower-case conversion characters except that the result is converted to upper case according to the rules of the prevailing Locale. The result is equivalent to the following invocation of String.toUpperCase()`

Reply:
I see now in the tests that this seems to be the correct implementation. Consider clarifying the comment.